src/expressServer.ts

```diff
@@ -76,6 +76,11 @@ export function setupEnvironment(): void {
 export type AddRoutes = (router: Router, options?: Partial<FernsRouterOptions<any>>) => void;
 
 const logRequestsFinished = function (req: any, res: any, startTime: [number, number]) {
+  const options = (req.loggingOptions ?? {}) as LoggingOptions;
+
+  const slowReadMs = options.logSlowRequestsReadMs ?? SLOW_READ_MAX;
+  const slowWriteMs = options.logSlowRequestsWriteMs ?? SLOW_WRITE_MAX;
+
   const diff = process.hrtime(startTime);
```

Comment:
Looks like hrtime is considered legacy now: https://nodejs.org/api/process.html#processhrtimetime

Reply:
Noted in a comment!